### PR TITLE
Document how to upgrade projects unattended

### PR DESCRIPTION
**Pull request by sofietoft**
Related to: https://github.com/umbraco/Umbraco-CMS/pull/9531 

Also included a "Table of contents" to the General Upgrades article.

### CODE REVIEW

**Review by rasmusjp — APPROVED** [review]
Looks good 💪 

We might want to add a note about all request made to the site will wait for the upgrade to finish before any content is returned. What do you think?

**Comment by sofietoft** [comment]
Yes. I think that's a great idea.
Should I added as part of the note about composers/components in the last part of the guide, or should it be a separate note somewhere else? 🤔

**Comment by rasmusjp** [comment]
Hmm... Good question 😄 Maybe add a new note after `With the correct configuration...` and then add a new headline before `The Runtime level will use...` called `Boot order` or something like that given it's a more technical explanation on how the boot process is working with unattended upgrades enabled.

File: Getting-Started/Setup/Upgrading/general.md [hunk 1]
```diff
@@ -83,12 +96,59 @@ The installer will do two things:
...
+
+With the correct configuration applied, the project will be upgraded on the next boot.
+
+:::note
+While the upgrade processes are running, any requests made to the site will be "put on hold", meaning that no content will be returned before the upgrade is complete.
+:::
+
+#### Boot order
+
+The Runtime level will use `Run` instead of `Upgrade` in order to allow the website to continue to boot up directly after the migration is run, instead of initiating the otherwise required restart.
```
Comment (sofietoft) [inline Getting-Started/Setup/Upgrading/general.md:138]: @rasmusjp - I've tried adding a note mentioning the requests to the site, and also added one more heading, to sorta split up this section a bit.

I know the phrasing in the new note is a bit strange, but I couldn't really come up with another way of saying it, without using "put on hold" 🤔 Any suggestions are very welcome!!
Let me know what you think!

**Pull request merged**